docker/stats/requirements.txt

```diff
@@ -6,6 +6,7 @@ wget
 requests
 odc-aws
 odc-aio
+odc-algo
```

Comment:
The issue is not missing `odc-algo` issue is missing `hdstats` which `odc-algo` is using without declaring dependency on it.